runtime/src/backends/dynamodb.rs

```diff
@@ -0,0 +1,448 @@
+use anyhow::Result;
+use async_trait::async_trait;
+use bytes::Bytes;
+use core::borrow::BorrowMut;
+use hyper::{HeaderMap, Response, StatusCode};
+use rusoto_core::{Region, RusotoError};
+use rusoto_dynamodb::{
+    AttributeValue, DeleteItemError, DeleteItemInput, DeleteItemOutput, DynamoDb, DynamoDbClient,
+    GetItemError, GetItemInput, GetItemOutput, PutItemError, PutItemInput, PutItemOutput,
+};
+use std::time::{Duration, SystemTime, UNIX_EPOCH};
+use std::{collections::HashMap, env};
+
+extern crate flexbuffers;
+
+use crate::storage::{Item, ItemKey, Storage};
+
+pub struct DynamoDbCacheClient {
+    connector: DynamoDbClient,
+    cache_key_name: String,
+    cache_resp_headers_name: String,
+    cache_resp_body_name: String,
+    cache_resp_status_name: String,
+    cache_surrogate_keys_name: String,
+    cache_creation_ts_name: String,
+    cache_expiry_ts_name: String,
+    sk_timestamp_name: String,
+    cache_table_name: String,
```

Comment:
Seems better to move this to config, but we can do it later.
We also can prepare default values.

Reply:
As discussed, we can revisit this when adjusting the configuration mechanism later.